--- node_modules/telegraf/lib/network/client.js	2017-09-22 04:26:12.556373950 +0200
+++ node_modules/telegraf/lib/network/client.js	2017-09-22 03:35:17.825691862 +0200
@@ -67 +67,2 @@
-    if (this.options.webhookReply && this.response && !this.response.finished && !isMultipart && !webhookBlacklist.includes(method)) {
+    const canUseWebhook = !(extra.noWebhook && delete extra.noWebhook)
+    if (canUseWebhook && this.options.webhookReply && this.response && !this.response.finished && !isMultipart && !webhookBlacklist.includes(method)) {
