--- node_modules/telegraf/core/network/client.js	2017-12-10 19:55:53.000000000 +0100
+++ node_modules/telegraf/core/network/client.js	2017-12-21 23:13:54.592558391 +0100
@@ -75 +75,2 @@
-    if (this.options.webhookReply && !isMultipartRequest && this.response && !this.response.finished && !WebhookBlacklist.includes(method)) {
+    const canUseWebhook = !(extra.noWebhook && delete extra.noWebhook)
+    if (canUseWebhook && this.options.webhookReply && !isMultipartRequest && this.response && !this.response.finished && !WebhookBlacklist.includes(method)) {
